guidance/models/_model.py

```diff
@@ -826,7 +826,8 @@ def _cleanup_tokens(self, token_ids, token_byte_positions):
                 pos += len(self.tokenizer.tokens[id])
                 token_byte_positions.append(pos)
 
-            # ugly hack to deal with sentence peice craziness of space hiding after special tokens TODO: figure out how to make this more robust
+            # ugly hack to deal with sentence piece craziness of space hiding after special tokens 
+            # TODO: figure out how to make this more robust
```

Comment:
Issue number on GitHub?

Reply:
I don't think there's a GH issue for this, just a bug we discovered early on.

Reply:
I'd rather just leave this one documented in the code, it's quite nuanced to type out in a GH issue and very contextually dependent on this specific function